lib/features/auth/presentation/components/login_form.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'file:///C:/Users/antho/Desktop/kabod_project/kabod_app/lib/core/presentation/res/constants.dart';
```

Comment:
Jean, note that, on my machine, I don't have a C:/Users/antho/ etc, etc, etc. The code doesn't work for me.
Also remember that 'C:/' is something from Windows machines. On Mac and Linux machines, paths are totally different (i.e. /users/antho/Desktop) so you have to import files on a different way to make sure it works on ANY machine or ANY mobile. Remember that your users' mobiles don't have your C/Users/antho directory also :P 

In order to fix that, you should use packages and relative paths.
For instance, for that import line, you would have use relative paths like this below:
```suggestion
import 'package:kabod_app/core/presentation/res/constants.dart';
```

If you are having trouble understanding imports, you can also press Alt+Enter on the unrecognized import and Android Studio will suggest you how to import it.

![image](https://user-images.githubusercontent.com/16261772/108627530-fcf2b380-744d-11eb-8c7d-2fdef2153611.png)

Reply:
got it.! this was a mistake, sometimes Android Studios create this path automatically.